R/utils.R

```diff
@@ -19,3 +19,17 @@ get_client_timezone <- function(ns) {
   shinyjs::runjs(script) # function does not return anything
   return(invisible(NULL))
 }
+
+#' Resolve the expected bootstrap version
+#' @keywords internal
+get_teal_bs_version <- function() {
```

Comment:
It would be good to support both R option as well as sys env. I would say that for the app users the latter one is more accessible through the config panel in RSConnect. That's true for each of our settings.
If added, please explain the order in the docs / vignette. A good example could be found here: https://rstudio.github.io/renv/reference/config.html#details

Reply:
actually it is true for only settings that could be expressed as a character

Reply:
We could not support sys env as we are expecting bslib::bs_theme which is quite complex object. sys env supports only base types like string.